eZ/Publish/Core/settings/repository.yml

```diff
@@ -1,161 +1,57 @@
+        factory: ["@ezpublish.api.repository", getURLAliasService]
 
     ezpublish.api.service.user:
-        class: "%ezpublish.api.service.user.class%"
-        factory: ["@ezpublish.api.repository.factory", buildService]
-        arguments: ["@ezpublish.api.repository", User]
-        lazy: true
+        class: eZ\Publish\API\Repository\UserService
+        factory: ["@ezpublish.api.repository", getUserService]
 
     ezpublish.api.service.search:
-        class: "%ezpublish.api.service.search.class%"
-        factory: ["@ezpublish.api.repository.factory", buildService]
-        arguments: ["@ezpublish.api.repository", Search]
-        lazy: true
+        class: eZ\Publish\API\Repository\SearchService
+        factory: ["@ezpublish.api.repository", getSearchService]
 
     ezpublish.api.service.section:
-        class: "%ezpublish.api.service.section.class%"
-        factory: ["@ezpublish.api.repository.factory", buildService]
-        arguments: ["@ezpublish.api.repository", Section]
-        lazy: true
+        class: eZ\Publish\API\Repository\SectionService
+        factory: ["@ezpublish.api.repository", getSectionService]
 
     ezpublish.api.service.trash:
-        class: "%ezpublish.api.service.trash.class%"
-        factory: ["@ezpublish.api.repository.factory", buildService]
-        arguments: ["@ezpublish.api.repository", Trash]
-        lazy: true
+        class: eZ\Publish\API\Repository\TrashService
+        factory: ["@ezpublish.api.repository", getTrashService]
 
     ezpublish.api.service.location:
-        class: "%ezpublish.api.service.location.class%"
-        factory: ["@ezpublish.api.repository.factory", buildService]
-        arguments: ["@ezpublish.api.repository", Location]
-        lazy: true
+        class: eZ\Publish\API\Repository\LocationService
+        factory: ["@ezpublish.api.repository", getLocationService]
 
     ezpublish.api.service.language:
-        class: "%ezpublish.api.service.language.class%"
-        factory: ["@ezpublish.api.repository.factory", buildService]
-        arguments: ["@ezpublish.api.repository", ContentLanguage]
-        lazy: true
-
-    # Signal Slot API wrapper
-    ezpublish.signalslot.repository:
-        class: "%ezpublish.signalslot.repository.class%"
-        arguments: ["@ezpublish.api.inner_repository", "@ezpublish.signalslot.signal_dispatcher_transaction_wrapper"]
-        lazy: true
-
-    ezpublish.signalslot.signal_dispatcher_transaction_wrapper:
-        class: "%ezpublish.signalslot.signal_dispatcher_transaction_wrapper.class%"
-        arguments: ["@ezpublish.signalslot.signal_dispatcher"]
-
-    ezpublish.signalslot.signal_dispatcher:
-        class: "%ezpublish.signalslot.signal_dispatcher.class%"
-        factory: ["@ezpublish.signalslot.signal_dispatcher.factory", buildSignalDispatcher]
+        class: eZ\Publish\API\Repository\LanguageService
+        factory: ["@ezpublish.api.repository", getLanguageService]
```

Comment:
I can't find anywhere `getLanguageService` method, so rather `getContentLanguageService` or refactor method name and tests to `getLanguageService`, right? (ref.: `\eZ\Publish\Core\SignalSlot\Repository`). 
Note: the actual class is called `LanguageService`, but I see that in inner repository we use `getContentLanguageService`, so not sure in which direction change should go.

Side note: this is quite unexpected, because I see behat tests pass, however locally when I go to `/ez` I get error related to this call.